src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,21 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <>
+    {comments.length > 0 ? (
+      <div className="CommentList">
+        {comments.map(comment => (
+          <CommentInfo comment={comment} key={comment.id} />
+        ))}
+      </div>
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings

Reply:
Components were written in advance by MateAcademy inside App.jsx.
The task was to split App.jsx by predefined Components (pre-created files).

Reply:
Done ✅